### DISCUSSION TITLE
rabbitmq.conf, advanced.config are not found under /etc/rabbitmq after RPM installation

### DISCUSSION THREAD

**Discussion by nvidiajunkie** [rabbitmq/rabbitmq-server#10013] [Other] [answered]
### Describe the bug

Yesterday I installed RMQ 3.12.10 and Erlang 26.1.2 via documented instructions found [here](https://live.rabbitmq.com/install-rpm.html) on the Rabbit website.  I navigated to the `/etc/rabbitmq` directory and only see `enable_plugins`.  Performing a `rabbitmq-diagnostics status` shows nothing within `Config files`. This same goes from the Management UI.

### Reproduction steps

1.Install RMQ+Erlang using https://live.rabbitmq.com/install-rpm.html documentation for RHEL 8
2. `ls /etc/rabbitmq` shows no config files
3. Install mlocate via dnf and `locate rabbitmq.conf` or `locate -i rabbitmq.conf`.  Nothing comes up for me
4. Restart host and revalidate steps 1-3
5. Remove Erlang and Rabbit with dnf and reinstall
6. Revalidate 1-3
7. Issue persists
8. `touch /etc/rabbitmq/rabbitmq.conf`, restart the service and service fails with `Error Code` but no code visible that I can see.
...

### Expected behavior

`rabbitmq.conf` and `advanced.config` are editable and can set / modify items like `ssl.options` and certificates used for the shovel URI paths.

### Additional context

I noticed this in 3.12.8 as well as the Docker image for 3.12.8 and now 3.12.10.

**Comment by michaelklishin** [comment, marked as answer]
@nvidiajunkie from the [Configuration guide](https://rabbitmq.com/configure.html#config-location):

```
Default configuration file location is distribution-specific. RabbitMQ packages or nodes will not create any configuration files. Users and deployment tool should use the following locations when creating the files
```

They are not "missing": they were never meant to be included. It's up to the operator (and the deployment tool used) to generate a suitable configuration file. There are examples with comments available from GitHub and linked to from the same doc guide. I would not recommend blindly copying it as the file is too large but you are welcome to copy and edit the relevant parts.

**Reply by nvidiajunkie** [reply]
Thank you sir for your quick reply. I looked for a _long_ time and can't believe I didn't see this.  I assumed when you add the management plugin and service started it would auto-gen it or something. Lesson learned. Cheers!
